addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1808,4 +1808,32 @@ options.registry.CoverProperties = options.Class.extend({
         this.$target[0].dataset.filterColor = activeFilterColor || '';
     },
 });
+
+/**
+ * Allows snippets to be moved before the preceding element or after the following
+ */
+options.registry.snippet_move = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Moves the snippet around
+     */
+    moveSnippet: function (previewMode, direction, $opt) {
```

Comment:
I'd prefer your keep the `value` variable name (easier when I have to refactor stuff).